tests/scripts/run-xunit-perf.cmd

```diff
@@ -21,6 +21,9 @@ setlocal
   set COLLECTION_FLAGS=stopwatch
   set ETW_COLLECTION=Off
   set STABILITY_PREFIX=
+  set BENCHVIEW_GROUP=CoreCLR
+  set HAS_WARMUP_RUN=
```

Comment:
I think that we should have this the other way around.  The default should be that we have a warmup run, and if you want to specify that you are not using one you should have to use a flag.